jenkins_pipelines/environments/uyuni-master-dev-acceptance-tests-PRV

```diff
@@ -10,7 +10,7 @@ node('sumaform-cucumber-provo') {
             string(name: 'cucumber_ref', defaultValue: 'master', description: 'Testsuite Git reference (branch, tag...)'),
             string(name: 'tf_file', defaultValue: 'susemanager-ci/terracumber_config/tf_files/Uyuni-Master-PRV.tf', description: 'Path to the tf file to be used'),
             string(name: 'sumaform_gitrepo', defaultValue: 'https://github.com/uyuni-project/sumaform.git', description: 'Sumaform Git Repository'),
-            string(name: 'sumaform_ref', defaultValue: 'master', description: 'Sumaform Git reference (branch, tag...)'),
+            string(name: 'sumaform_ref', defaultValue: 'rename_cucumber_testsuite_module', description: 'Sumaform Git reference (branch, tag...)'),
```

Comment:
is this a leftover of your tests, or going to stay?

Reply:
It's going to stay for now. I need to rename all the pipelines before merging this cucumber testsuite module. I will switch back to master once done.